code/_core/obj/item/container/food/package/conserved.dm

```diff
@@ -0,0 +1,106 @@
+/obj/item/container/food/package/conserved
+	icon = 'icons/obj/item/consumable/food/conserved.dmi'
+	packaging = null
```

Comment:
It might need some other way of implementation, and by this I mean something that's not equal to null.
Couldn't think of anything else, so, yes.
:thinking: Maybe a 'nothing' item could work.